### PR TITLE
[no-issue] ParsingFailed, Notification

### PR DESCRIPTION
**Pull request by avchu**

### CODE REVIEW

File: worker/src/main/scala/com/github/jaitl/crawler/worker/pipeline/PipelineBuilder.scala [hunk 1]
```diff
@@ -24,6 +25,11 @@ private[pipeline] class PipelineBuilder[T] {
     this
   }
 
+  def withEmailNotifier(emailImpl: BaseNotification): this.type = {
```
Comment (Jaitl) [inline worker/src/main/scala/com/github/jaitl/crawler/worker/pipeline/PipelineBuilder.scala]: Email only?

Comment (avchu) [reply worker/src/main/scala/com/github/jaitl/crawler/worker/pipeline/PipelineBuilder.scala]: Fixed )

File: worker/src/main/scala/com/github/jaitl/crawler/worker/executor/TasksBatchController.scala [hunk 2]
```diff
@@ -64,6 +68,7 @@ private[worker] class TasksBatchController(
   val taskQueue: mutable.Queue[QueuedTask] = mutable.Queue.apply(batch.tasks.map(QueuedTask(_, 0)): _*)
 
   private var resourceController: ActorRef = _
+  private var emailNotifier: ActorRef = _
```
Comment (Jaitl) [inline worker/src/main/scala/com/github/jaitl/crawler/worker/executor/TasksBatchController.scala]: email

File: worker/src/main/scala/com/github/jaitl/crawler/worker/executor/TasksBatchController.scala [hunk 3]
```diff
@@ -74,6 +79,7 @@ private[worker] class TasksBatchController(
     resourceController = resourceControllerCreator.create(this.context, configPipeline.resourceType)
     saveCrawlResultController = saveCrawlResultCreator.create(this.context, pipeline, self, configPipeline)
     crawlExecutor = crawlExecutorCreator.create(this.context)
+    emailNotifier = notifierExecutorCreator.create(this.context)
```
Comment (Jaitl) [inline worker/src/main/scala/com/github/jaitl/crawler/worker/executor/TasksBatchController.scala]: and here email

**Pull request merged**